src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -352,6 +352,31 @@ public static FormulaValue Right(IRContext irContext, FormulaValue[] args)
             return new StringValue(irContext, source.Value.Substring(source.Value.Length - (int)count.Value, (int)count.Value));
         }
 
+        private static FormulaValue Find(IRContext irContext, FormulaValue[] args)
+        {
+            var findText = (StringValue)args[0];
+            var withinText = (StringValue)args[1];
+
+            NumberValue startIndex;
+            if (args[2] is BlankValue)
```

Comment:
This should be taken care of in the wrapper. See the above comment. We avoid doing checks for blank in the body of functions.

Reply:
return blank if arg is blank only works for arg[2] (startIndex)...

Reply:
fixed